src/stories/containers/EndgameBudgetContainerSecondLevel/useEndgameBudgetContainerSecondLevel.tsx

```diff
@@ -43,20 +43,31 @@ export const useEndgameBudgetContainerSecondLevel = (budgets: Budget[]) => {
     handlePeriodChange,
     periodicSelectionFilter,
     handleSelectChangeMetrics,
-  } = useBreakdownTable();
+  } = useBreakdownTable(initialYear);
 
   const { headersExpenseReport, reportExpenseItems, handleLoadMore, showSome, onSortClick } =
     useDelegateExpenseTrendFinances();
 
-  const levelBudget = budgets?.find((budget) => budget.codePath === levelCode);
+  const levelBudget = budgets.find((budget) => budget.codePath === levelPath);
   const title = removePrefix(levelBudget?.name || '', prefixToRemove) || '';
   const { isLight } = useThemeContext();
-  const [year, setYears] = useState<string>('2022');
+  const [year, setYear] = useState(initialYear);
   const isMobile = useMediaQuery(lightTheme.breakpoints.down('tablet_768'));
   const icon = levelBudget?.image || '';
   // const icon = itemTitle?.image || '';
   const handleChangeYearsEndgameAtlasBudget = (value: string) => {
-    setYears(value);
+    setYear(value);
+    router.push(
+      {
+        pathname: '/finances/[firstPath]',
```

Comment:
Please, use `siteRoutes` instead to genereta routes

Reply:
Done! check: https://github.com/makerdao-ses/ecosystem-dashboard/pull/1117/commits/dcb1808c34b8389cc547be03221a06f17d27209d